modules/lvgl/Kconfig

```diff
@@ -53,6 +54,29 @@ endchoice
 config LV_COLOR_16_SWAP
 	bool
 
+config LV_Z_FLUSH_THREAD
+	bool "Flush LVGL frames in a separate thread"
+	help
```

Comment:
Why wouldn't we want to set a default true here for this option?

Reply:
This was asked in the original PR here https://github.com/zephyrproject-rtos/zephyr/pull/59166#issuecomment-1590193173